test/e2e/run_networking_test.go

```diff
@@ -1186,4 +1186,21 @@ EXPOSE 2004-2005/tcp`, ALPINE)
 		Expect(session).Should(Exit(0))
 		Expect(session.OutputToStringArray()).To(HaveLen(4), "output should only show link local address")
 	})
+
+	It("podman run with macvlan network", func() {
+		net := "mv-" + stringid.GenerateRandomID()
+		session := podmanTest.Podman([]string{"network", "create", "-d", "macvlan", "--subnet", "10.10.0.0/24", net})
+		session.WaitWithDefaultTimeout()
+		defer podmanTest.removeNetwork(net)
```

Comment:
Nit: move this line below the following exit-code check.  If creation fails, remove will as well and cause some noise in the output.

Reply:
this pattern is used almost everywhere I rather not change that now

Reply:
That makes many small nits :^)